src/containerapp/azext_containerapp/_params.py

```diff
@@ -63,6 +63,15 @@ def load_arguments(self, _):
     with self.argument_context('containerapp env', arg_group='Monitoring') as c:
         c.argument('logs_dynamic_json_columns', options_list=['--logs-dynamic-json-columns', '-j'], arg_type=get_three_state_flag(),
                    help='Boolean indicating whether to parse json string log into dynamic json columns. Only work for destination log-analytics.', is_preview=True)
+        
+    # Telemetry 
+    with self.argument_context('containerapp env telemetry set', arg_group='Telemetry') as c:
```

Comment:
Did you test command `az containerapp env telemetry set -h`? 
As `az containerapp env telemetry` is a preview command, no need to add `is_preview` tag in params, otherwise the `--help` result with looks strange.